tests/PHPStan/Rules/Functions/DefineParametersRuleTest.php

```diff
@@ -0,0 +1,31 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Rules\Functions;
+
+use PHPStan\Php\PhpVersion;
+
+/**
+ * @extends \PHPStan\Testing\RuleTestCase<DefineParametersRule>
+ */
+class DefineParametersRuleTest extends \PHPStan\Testing\RuleTestCase
+{
+
+	protected function getRule(): \PHPStan\Rules\Rule
+	{
+		return new DefineParametersRule(new PhpVersion(PHP_VERSION_ID));
+	}
+
+	public function testFile(): void
+	{
+		if (PHP_VERSION_ID < 80000) {
+			$this->markTestSkipped('Test requires PHP 8.0.');
```

Comment:
On lower versions we can test that it doesn't report anything even for the three-argument call.